### ISSUE TITLE
[BUG] v0.7.0 dropped python 3.8 ?

### ISSUE THREAD

**Issue by rcaneill** [xgcm/xgcm#484]
**What happened**:

I was unable to install xgcm version 0.7.0 with python 3.8

**What you expected to happen**:
As xarray is still supporting python 3.8, I was expecting to be able to install xgcm 0.7.0

As it is not written in the what's new, I guess that dropping python 3.8 may be an error? (or otherwise, it should probably be written in the what's new)

**Comment by jbusecke** [comment]
Yes we had to drop 3.8 for the new changes. But we should probably ammend the release notes and whats new!

**Comment by rabernat** [comment]
Thanks for reporting @rcaneill!

Is dropping 3.8 support a problem for you? We made this decision in order to take advantage of some great language features in 3.9 (see discussion here https://github.com/xgcm/xgcm/pull/418#pullrequestreview-873061067). But perhaps we were too hasty and did not consult widely enough.

**Comment by rabernat** [comment]
Good catch Anderson. The dependency is now >= 3.9, so it looks like we need to rebuild the recipe.

**Comment by rcaneill** [comment]
Dropping 3.8 is not a problem for me, I was just surprise that the support disappeared without any notice.
